Update blob-inventory-how-to.md (#120636)
Azure/azure-content: articles/storage/blobs/blob-inventory-how-to.md

5. In the **Add a rule** page, name your new rule.

> 6. Choose a container. This container act as the target location for generation of reports.

Reviewer: ```suggestion
6. Choose the container that will store inventory reports.
```
